packages/type-utils/src/isTypeReadonly.ts

```diff
@@ -22,32 +23,63 @@ const enum Readonlyness {
 
 export interface ReadonlynessOptions {
   readonly treatMethodsAsReadonly?: boolean;
+  readonly allowlist?: Array<TypeAllowlistItem>;
 }
 
-export const readonlynessOptionsSchema = {
```

Comment:
Removing this would be a breaking change as it is publicly exported

Reply:
Ok, I'll add it back in...